src/libOpenImageIO/imagebufalgo.cpp

```diff
@@ -927,7 +926,7 @@ ImageBufAlgo::fillholes_pushpull (ImageBuf &dst, const ImageBuf &src,
     topspec.set_format (TypeDesc::FLOAT);
     ImageBuf *top = new ImageBuf (topspec);
     paste (*top, topspec.x, topspec.y, topspec.z, 0, src);
-    pyramid.push_back (boost::shared_ptr<ImageBuf>(top));
+    pyramid.push_back (OIIO::shared_ptr<ImageBuf>(top));
```

Comment:
Yes, eventually. Can't be done yet, without either a proliferation of "#if" or making a solid C++03 implementation of make_shared.